_posts/2021-10-04-sharing-materials-from-ncwit-talk.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Sharing resources from recent NCWIT talk on counteracting bias
+tags: [events, DEI, working groups]
+posted_by: David Nicholson
+---
+
+On Thursday, August 12th, US-RSE held the
+[second talk of our speaker series](https://us-rse.org/events/2021/2021-07-dei-speaker-series-ncwit)
+organized by the Diversity, Equity, and Inclusion (DEI) working group.
+We were joined by [Joanne Esch](https://ncwit.org/profile/joanne-esch/)
+and [Andrea Bowen-Jones](https://ncwit.org/profile/andrea-bowens-jones/)
+from [The National Center for Women & Information Technology (NCWIT)](https://ncwit.org/).
+They spoke with us the biases and barriers that
```

Comment:
```suggestion
They spoke with us about the biases and barriers that
```
(or similar)

Reply:
Thank you! I managed to introduce some new "bugs" revising :(
Fixed